### PR TITLE
Release v1.30

### PR DESCRIPTION
**Pull request by bradchiappetta**
- Move to more standardized logging practices for better third party usage
- Update Quick command to use POST endpoint instead of GET endpoint

Fixes:
#647 

Includes:
#627 #628 #630 #634 #645 #646 #648 #650

### CODE REVIEW

**Comment by guzzijones** [comment]
+1 from me.  logging is now better fit for use as a library.

File: src/greynoise/api/__init__.py [hunk 1]
```diff
@@ -325,13 +314,7 @@ def query(self, query, size=None, scroll=None):
...
             response = {"message": "GNQL not supported with Community offering"}
         else:
-            LOGGER.debug(
-                "Running GNQL query: %s...",
-                query,
-                query=query,
-                size=size,
-                scroll=scroll,
-            )
+            LOGGER.debug("Running GNQL query: %s % s %s...", query, size, scroll)
```
Comment (ianling) [inline src/greynoise/api/__init__.py:317]: Extra space

```suggestion
            LOGGER.debug("Running GNQL query: %s %s %s...", query, size, scroll)
```

**Review by ianling — APPROVED** [review]
Only saw the one small typo, everything looks good otherwise!

File: src/greynoise/api/__init__.py (hunk 1 above)
Comment (bradchiappetta) [reply src/greynoise/api/__init__.py]: Fixed pushed.

**Pull request merged**